app/views/admin/pages/_form.html.erb

```diff
@@ -52,7 +52,7 @@
       <p>Determines if it is accessible by <code>/page-slug</code> vs <code>/page/page-slug</code> Be careful! ⚠️</p>
     </div>
 
-    <% if ForemInstance.private? %>
+    <% if !Settings::UserExperience.public? %>
```

Comment:
@juliannatetreault I think we should update the definition of `ForemInstance.private?` instead. The current one is wrong and could be renamed to `ForemInstance.invitation_only?`.

Reply:
Ah, gotcha -- I think I may have misunderstood the direction I needed to take here. Will do!